modules/ztp-image-based-upgrade-procedure-steps.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-procedure-steps_{context}"]
+= Performing an image-based upgrade on managed clusters at scale in several steps
+
+For use cases when you need better control of when the upgrade interrupts your service, you can upgrade a set of your managed clusters by using the `ImageBasedGroupUpgrade` CR with adding actions after the previous step is complete.
```

Comment:
"previous step" might be tricky. What is the previous step. Best to call it out to avoid confusion.

Reply:
It's always a different step, so not sure how to better phrase it